Source/JavaScriptCore/runtime/AbstractModuleRecord.cpp

```diff
@@ -858,7 +858,7 @@ static String printableName(const RefPtr<UniquedStringImpl>& uid)
 {
     if (uid->isSymbol())
         return uid.get();
-    return WTF::makeString("'", String(uid.get()), "'");
+    return WTF::makeString('\'', String(uid.get()), '\'');
```

Comment:
Should be able to pass `*uid` or `uid.get()` without calling `String` and avoid the reference count churn.